ROMFS/px4fmu_common/init.d/airframes/50003_aion_robotics_r1_rover

```diff
@@ -46,15 +48,15 @@ param set-default GND_SPEED_THR_SC 1
 
 param set-default NAV_ACC_RAD 0.5
 
-# Differential drive acts like ackermann steering with a maximum turn angle of 180 degrees, or pi radians
-param set-default GND_MAX_ANG 3.1415
-
 # Set geometry & output configration
 param set-default CA_AIRFRAME 6
 param set-default CA_R_REV 3
-param set-default PWM_MAIN_FUNC1 101
-param set-default PWM_MAIN_FUNC2 102
-param set-default PWM_MAIN_DIS1 1500
-param set-default PWM_MAIN_DIS2 1500
-param set-default PWM_MAIN_TIM0 50
-param set-default PWM_MAIN_TIM1 50
+
+
+param set-default RBCLW_ADDRESS 130
+param set-default RBCLW_SER_CFG 202
+param set-default ROBOCLAW_FUNC1 101
+param set-default ROBOCLAW_FUNC2 102
+param set-default ROBOCLAW_REV 1
+param set-default SENS_EN_INA226 0
+param set-default SER_GPS2_BAUD 57600
```

Comment:
Same here, this is port and configuration-specific. We should document how to set this correctly.

Reply:
Will start working on the documentation now.